### DISCUSSION TITLE
模仿游戏内鸣神大社求签功能，修改prophecy.js

### DISCUSSION THREAD

**Discussion by QyInvoLing** [Arondight/Adachi-BOT#440] [Show and tell]
将第四行的种子改为const seed = Math.round(Date.now() / 86400000 + msg.uid);即可让每个用户每天求签的结果相同
（水）

**Comment by Arondight** [comment]
这个是故意每次都不同的，不过数据都在别人的服务器上，我感觉什么时候需要挂个脚本都扒下来

**Comment by mark9804** [comment]
求签功能其实我[有想法写一个味儿更冲的文案](https://github.com/Mark9804/Adachi-BOT/issues/15)，只是最近事情太多了，一直没动

**Reply by Arondight** [reply]
哈哈哈，我个人更喜欢当前和游戏无关的这个，比较接地气
